src/components/Link/index.tsx

```diff
@@ -77,7 +77,7 @@ const ResultLinkComponent: React.FC<ILinkProps> = ({
 }
 
 const scrollToHash = (hash: string, scrollOptions = {}): void => {
-  if (hash) {
+  if (hash && /^#[A-Za-z][-A-Za-z0-9_:.]*$/.test(hash)) {
```

Comment:
Whenever an anchor link didn't match a valid HTML id, querySelector through an error and crashed the site. I added a regex test stopping querySelector from running unless the anchor link is a valid id.